squad_client/core/models.py

```diff
@@ -40,13 +40,9 @@ def __fill__(self, klass, results):
         objects = {}
         for result in results:
             obj = klass()
-            attrs = klass.attrs if len(klass.attrs) else [attr.replace(' ', '_') for attr in result.keys()]
+            attrs = klass.attrs if len(klass.attrs) else [attr for attr in result.keys()]
```

Comment:
LGTM. True. @chaws and I noticed some values weren't collected like 'build-location'..etc.

Reply:
@mwasilew this will fail for some "metadata" fields. They don't have a standard naming convention, so it can vary from 'the-field' or 'the/field', which will raise an error when python tries to make an instance attribute out of the name.

Reply:
I didn't see that in case of android. Should I just add '-' and '/' to the replace command (and maybe a unit test)?

Reply:
Adding the extra '-' and '/' would be better.

Reply:
ok, done. I guess metadata can include some more characters that python won't like (:;,*\ etc.)